tests/imports.pyi

```diff
@@ -93,7 +93,7 @@ from typing import ContextManager  # Y022 Use "contextlib.AbstractContextManager
 from typing import OrderedDict  # Y022 Use "collections.OrderedDict[KeyType, ValueType]" instead of "typing.OrderedDict[KeyType, ValueType]" (PEP 585 syntax)
 from typing_extensions import OrderedDict  # Y022 Use "collections.OrderedDict[KeyType, ValueType]" instead of "typing_extensions.OrderedDict[KeyType, ValueType]" (PEP 585 syntax)
 from typing import Callable  # Y022 Use "collections.abc.Callable" instead of "typing.Callable" (PEP 585 syntax)
-from typing import ByteString  # Y022 Use "collections.abc.ByteString" instead of "typing.ByteString" (PEP 585 syntax)
+from typing import ByteString
```

Comment:
Nit: can we move this out of the "BAD IMPORTS" section? (It should probably go on line 46)